src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/AvatarImage.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources
+    xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="Sylius\Component\Core\Model\AvatarImage" uriTemplate="/admin/administrators/{id}/avatar-image">
+        <uriVariables>
+            <uriVariable parameterName="id" fromClass="Sylius\Component\Core\Model\AdminUser" fromProperty="avatar"/>
+        </uriVariables>
+
+        <validationContext>
+            <values>
+                <value name="groups">
+                    <values>
+                        <value>sylius</value>
+                    </values>
+                </value>
+            </values>
+        </validationContext>
+
+        <operations>
+            <operation class="ApiPlatform\Metadata\Get">
+                <normalizationContext>
+                    <values>
+                        <value name="groups">
+                            <values>
+                                <value>admin:avatar_image:read</value>
```

Comment:
Following notation of **api request - serialization group**, then it should contain `admin_user` / `administrator` somehow, as `{id}` is of admin user.

```patch
--- <value>admin:avatar_image:read</value>
+++ <value>admin:admin_user:avatar_image:read</value>
```